apps/teams-test-app/src/public/externalOauth_end.html

```diff
@@ -1,29 +1,40 @@
-<!DOCTYPE html>
 <html>
   <head>
+    <script
+      type="text/javascript"
+      src="
+    https://res.cdn.office.net/teams-js/2.21.0/js/MicrosoftTeams.min.js"
+    ></script>
     <h2 style="font-size: 40px">Starting mock external oauth2</h2>
-  </head>
-  <body>
     <script>
-      const failureMessage = {
-        type: 'authenticationFailed',
-        data: 'error',
-      };
-      const successMessage = {
-        type: 'authenticationSuccess',
-        data: 'success',
-      };
+      function appendLog(msg) {
```

Comment:
nit: why does it say appendLog? when you are rewriting the value everytime?

Reply:
Now it is append basic lines.